charts/tusd/templates/pdb.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "tusd.fullname" . }}
+  labels:
+    {{- include "tusd.labels" . | nindent 4 }}
+spec:
+  {{- with .Values.podDisruptionBudget.minAvailable }}
+  minAvailable: {{ . }}
+  {{- end }}
+  {{- with .Values.podDisruptionBudget.maxUnavailable }}
+  maxUnavailable: {{ . }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "tusd.selectorLabels" . | nindent 6 }}
```

Comment:
Looks like `{{- end }}` is missing from the end of the file (the initial if is not terminated)